Gopkg.lock

```diff
@@ -2,21 +2,20 @@
 
 
 [[projects]]
-  digest = "1:7131fa3a68e56764067bb569886cc7b03de5523fe5e81cbe0c70368932a1c622"
+  digest = "1:b24249f5a5e6fbe1eddc94b25973172339ccabeadef4779274f3ed0167c18812"
   name = "cloud.google.com/go"
   packages = [
     "compute/metadata",
     "iam",
     "internal",
     "internal/optional",
-    "internal/trace",
     "internal/version",
     "storage",
     "trace/apiv1",
   ]
   pruneopts = ""
-  revision = "28a4bc8c44b3acbcc482cff0cdf7de29a4688b61"
-  version = "v0.35.1"
+  revision = "2d3a6656c17a60b0815b7e06ab0be04eacb6e613"
+  version = "v0.16.0"
```

Comment:
Is this an upgrade? O.o

Reply:
We need to dive into it, why.. This is really bad, maybe it is just a go dep problem?

Reply:
so actually our `toml` is saying `0.16.0` (but with bug: `0.16.0` -> `v0.16.0`). Will fix this and rebase this PR